pennylane_lightning_kokkos/src/tests/TestHelpers.hpp

```diff
@@ -8,6 +8,88 @@ namespace {
 using namespace Pennylane;
 }
 
+namespace Pennylane {
+template <class T, class Alloc = std::allocator<T>> struct PLApprox {
```

Comment:
Could you please add documentation for PLApprox?

Reply:
done!